rspec-tools/rspec_tools/utils.py

```diff
@@ -77,6 +78,7 @@
     'objectivec': 'objectivec',
     'vb': 'vb',
     # these languages are not supported by highlight.js as the moment:
+    'ansible': 'ansible',
```

Comment:
```suggestion
    'objectivec': 'objectivec',
    'vb': 'vb',
     'ansible': 'yaml',
    # these languages are not supported by highlight.js as the moment:
```
Let's do the sane as kubernetes here.